src/actions/entries.js

```diff
@@ -1 +1,38 @@
-export const addEntry = (week) => {};
+import { SET_ENTRIES } from "./TYPES";
+import axios from "axios";
+
+const BASE_URL = "https://vip3e81bu0.execute-api.us-east-1.amazonaws.com/test";
+
+export const getEntries = (cognitoId, jwtToken, weekId) => async (dispatch) => {
+  try {
+    const BEARER_TOKEN = "Bearer " + jwtToken;
+    const headers = { headers: { Authorization: BEARER_TOKEN } };
+    const path = BASE_URL + `/users/${cognitoId}/journal/weeks/${weekId}`;
+    const res = await axios.get(path, headers);
+    dispatch({
+      type: SET_ENTRIES,
+      payload: res.data.Item.docBody.journal.weeks[0].entries,
+    });
```

Comment:
nothing wrong, but I think you might wanna add an item to the backlog to make a `Take Notes API` class that handles sending requests to our API. Will definitely make code more scalable. But I think go with this for now and add that to the backlog.

Reply:
yeah agree with this - I might make a ticket to do this